src/Http/Http.Results/src/Results.cs

```diff
@@ -0,0 +1,1243 @@
+        /// <param name="contentType">The Content-Type of the file.</param>
+        /// <param name="enableRangeProcessing">Set to <c>true</c> to enable range requests processing.</param>
+        /// <returns>The created <see cref="PhysicalFileResult"/> for the response.</returns>
+        public static IResult PhysicalFile(string physicalPath, string contentType, bool enableRangeProcessing)
+            => PhysicalFile(physicalPath, contentType, fileDownloadName: null, enableRangeProcessing: enableRangeProcessing);
+
+        /// <summary>
+        /// Returns the file specified by <paramref name="physicalPath" /> (<see cref="StatusCodes.Status200OK"/>) with the
+        /// specified <paramref name="contentType" /> as the Content-Type and the
+        /// specified <paramref name="fileDownloadName" /> as the suggested file name.
+        /// This supports range requests (<see cref="StatusCodes.Status206PartialContent"/> or
+        /// <see cref="StatusCodes.Status416RangeNotSatisfiable"/> if the range is not satisfiable).
+        /// </summary>
+        /// <param name="physicalPath">The path to the file. The path must be an absolute path.</param>
+        /// <param name="contentType">The Content-Type of the file.</param>
+        /// <param name="fileDownloadName">The suggested file name.</param>
+        /// <returns>The created <see cref="PhysicalFileResult"/> for the response.</returns>
+        public static IResult PhysicalFile(
+            string physicalPath,
+            string contentType,
+            string? fileDownloadName)
+            => new PhysicalFileResult(physicalPath, contentType) { FileDownloadName = fileDownloadName };
+
+        /// <summary>
+        /// Returns the file specified by <paramref name="physicalPath" /> (<see cref="StatusCodes.Status200OK"/>) with the
+        /// specified <paramref name="contentType" /> as the Content-Type and the
+        /// specified <paramref name="fileDownloadName" /> as the suggested file name.
+        /// This supports range requests (<see cref="StatusCodes.Status206PartialContent"/> or
+        /// <see cref="StatusCodes.Status416RangeNotSatisfiable"/> if the range is not satisfiable).
+        /// </summary>
+        /// <param name="physicalPath">The path to the file. The path must be an absolute path.</param>
+        /// <param name="contentType">The Content-Type of the file.</param>
+        /// <param name="fileDownloadName">The suggested file name.</param>
+        /// <param name="enableRangeProcessing">Set to <c>true</c> to enable range requests processing.</param>
+        /// <returns>The created <see cref="PhysicalFileResult"/> for the response.</returns>
+        public static IResult PhysicalFile(
+            string physicalPath,
+            string contentType,
+            string? fileDownloadName,
+            bool enableRangeProcessing)
+            => new PhysicalFileResult(physicalPath, contentType)
```

Comment:
Curious, why is this one expression bodied but the next one not?

Reply:
Somebody forgot to update MVC: https://github.com/dotnet/aspnetcore/blob/main/src/Mvc/Mvc.Core/src/ControllerBase.cs#L1647-L1677